webapi/src/test/scala/org/knora/webapi/store/triplestore/api/TriplestoreServiceInMemory.scala

```diff
@@ -246,11 +261,11 @@ final case class TriplestoreServiceInMemory(datasetRef: Ref[Dataset], implicit v
   override def getDataset: UIO[Dataset] =
     datasetRef.get
 
-  override def printDataset: UIO[Unit] =
+  override def printDataset(prefix: String = ""): UIO[Unit] =
     for {
-      _  <- Console.printLine(s"TDB Context:\n${TDB.getContext}\n").orDie
+//      _  <- Console.printLine(s"TDB Context:\n${TDB.getContext}\n").orDie
       ds <- getDataset
-      _  <- Console.printLine(s"TriplestoreServiceInMemory.printDataset:").orDie
+      _  <- Console.printLine(s"${prefix}TriplestoreServiceInMemory.printDataset:").orDie
```

Comment:
@BalduinLandolt added it there and I guess it was helpful for him. 
When I was running the tests it was not helpful for my use case. I left it so that when running the tests it is easy to add this printLine should you need it.